### PR TITLE
Update VVL for the 1.1.130 Vulkan Headers

### PR DESCRIPTION
**Pull request by mark-lunarg**
- Updated known_good json files for desktop and Android
- Fixed some old VUID-related issues
- Removed the obsolete old-to-new VUID mapping file
- Updated for VUID changes/deletions:
- Deleted VUIDs:
    "VUID-VkPipelineExecutableInternalRepresentationKHR-name-parameter"
    "VUID-VkPipelineExecutableInternalRepresentationKHR-description-parameter"
    "VUID-VkPipelineExecutableInternalRepresentationKHR-pData-parameter"
    "VUID-vkCmdClearDepthStencilImage-aspectMask-02499"
    "VUID-vkCmdClearDepthStencilImage-aspectMask-02500"		  
    "VUID-VkDeviceQueueInfo2-flags-requiredbitmask"
- New VUIDs:
    "VUID-VkAttachmentDescription-format-03280"
    "VUID-VkAttachmentDescription-format-03281"
    "VUID-VkAttachmentDescription-format-03282"
    "VUID-VkAttachmentDescription-format-03283"
    "VUID-VkAttachmentDescription2KHR-format-03294"
    "VUID-VkAttachmentDescription2KHR-format-03295"
    "VUID-VkAttachmentDescription2KHR-format-03296"
    "VUID-VkAttachmentDescription2KHR-format-03297"
    "VUID-VkAttachmentReference2KHR-layout-03077"
    "VUID-VkAttachmentReference2KHR-attachment-03311"
    "VUID-VkAttachmentReference2KHR-attachment-03312"
    "VUID-vkGetQueryPoolResults-flags-02828"
    "VUID-vkCmdClearDepthStencilImage-image-02825"
    "VUID-vkGetQueryPoolResults-flags-02827"
    "VUID-vkGetPhysicalDeviceToolPropertiesEXT-physicalDevice-parameter"
    "VUID-vkGetPhysicalDeviceToolPropertiesEXT-pToolCount-parameter"
    "VUID-vkGetPhysicalDeviceToolPropertiesEXT-pToolProperties-parameter"
    "VUID-vkGetPhysicalDeviceToolPropertiesEXT-pToolCount-arraylength"
    "VUID-VkPhysicalDeviceToolPropertiesEXT-sType-sType"
    "VUID-VkPhysicalDeviceToolPropertiesEXT-pNext-pNext"

### CODE REVIEW

File: layers/core_validation.cpp [hunk 1]
```diff
@@ -3277,7 +3277,7 @@ bool CoreChecks::PreCallValidateGetQueryPoolResults(VkDevice device, VkQueryPool
                                                     VkQueryResultFlags flags) const {
     if (disabled.query_validation) return false;
     bool skip = false;
-    skip |= ValidateQueryPoolStride("VUID-vkGetQueryPoolResults-flags-00814", "VUID-vkGetQueryPoolResults-flags-00815", stride,
+    skip |= ValidateQueryPoolStride("VUID-vkGetQueryPoolResults-flags-02827", "VUID-vkGetQueryPoolResults-flags-00815", stride,
```
Comment (mikes-lunarg) [inline layers/core_validation.cpp]: The VUIDs in this area are not great, but I think `ValidateQueryPoolStride` should probably be skipped if the query pool was created with type `VK_QUERY_TYPE_PERFORMANCE_QUERY`.  VUID-02828 might be a better fit, but again, the existing VUIDs step on each other so it requires a bit of interpretation.

File: layers/old_vuid_to_new_mapping.txt [hunk 2]
```diff
@@ -1,3107 +0,0 @@
-# This doc is stored for historical purposes
```
Comment (mikes-lunarg) [inline layers/old_vuid_to_new_mapping.txt]: sweet!

File: layers/core_validation.cpp (hunk 1 above)
Comment (mark-lunarg) [reply layers/core_validation.cpp]: I'll handle this in a follow up PR that will also implement 03229.

File: tests/vkpositivelayertests.cpp [hunk 3]
```diff
@@ -7950,7 +7950,7 @@ TEST_F(VkPositiveLayerTest, TransferImageToSwapchainDeviceGroup) {
 
 #if defined(VK_USE_PLATFORM_ANDROID_KHR)
     printf(
-        "%s According to VUID-01631, VkBindImageMemoryInfo-memory should be NULL. But Android will crash if memory is NULL, "
```
[the remark this answers is not in the document]
Comment (mark-lunarg) [reply tests/vkpositivelayertests.cpp]: Fixed, excellent catch.

**Comment by mark-lunarg** [comment]
Pushed to Master.

**Pull request closed without merging**